task/sonatype/0.1/README.md

```diff
@@ -0,0 +1,43 @@
+# Sonatype
+
+Contains all the tasks which use the Sonatype Nexus Lifecycle CLI.
+
+## Tasks
+
+This is a description of all the tasks, along with parameters, and installation instructions.
+
+- Neuxs Lifecycle Scan Task (nexus-lifecycle-scan-task.yml)
+
+----
+
+### Neuxs Lifecycle Scan Task (nexus-lifecycle-scan-task.yml)
+
+Invokes a Nexus Lifecycle scan
+
+#### Install Task
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/sonatype/0.1/nexus-lifecycle-scan/nexus-lifecycle-scan-task.yaml
+```
+
+#### Parameters
+
+The cli has it's own default parameters.  This task inherets the default parameters for the CLI.  To learn more about the default parameters, visit the cli documentation at [Nexus IQ CLI Docs](https://hub.docker.com/r/sonatype/nexus-iq-cli).
+
+name                        | description                                                                                                   | required  | default
+---------                   | -------------------------------------------                                                                   | --------  | -------
+SERVER_URL                  | URL to the IQ Server that will evaluate policies                                                              |     x     |
+AUTHENTICATION              | Authentication credentials to use for the IQ Server                                                           |     x     |
+APPLICATION_ID              | Public ID of the application on the IQ Server                                                                 |     x     |
+STAGE                       | The stage to run analysis against. Accepted values: [develop, build, stage-release, release, operate]         |     x     | [See CLI Default](https://hub.docker.com/r/sonatype/nexus-iq-cli)
```

Comment:
I think STAGE is not required. It will default to `develop` if it is not provided.